tests/unit_tests/block_reward.cpp

```diff
@@ -47,7 +47,7 @@ namespace
   };
 
   #define TEST_ALREADY_GENERATED_COINS(already_generated_coins, expected_reward)                              \
-    m_block_not_too_big = get_block_reward(0, current_block_weight, already_generated_coins, m_block_reward,1); \
+    m_block_not_too_big = get_block_reward(0, current_block_weight, already_generated_coins, m_block_reward, 1, 1); \
```

Comment:
Why did  you change these? based on what repository, commit, etc

Reply:
Based on: https://github.com/ombre-projects/ombre/blob/ebb16d452fccafbdbdac5ce7995c7730f34d44c6/tests/unit_tests/block_reward.cpp